docs/UserGuide.md

```diff
@@ -318,12 +318,13 @@ You can use this command to reset Tutor's Pet and all data will be cleared.
 **Format**: `reset`
 
 For example:
-* It is the start of a new semester and you want to delete all students and classes.
-  You can type in the command `reset` and press <kbd>Enter</kbd>.
+1. It is the start of a new semester and you want to delete all students and classes.
+
+1. You can type in the command `reset` and press <kbd>Enter</kbd>.
 
   ![Reset before](images/ugimages/ResetBefore.png)
 
-* You can see that all students and classes have been deleted.
+1. You can see that all students and classes have been deleted.
```

Comment:
Checked the ug on your branch. The steps counter will reset to 1 if there isn't an additional space at image of line 324/325.